components/NodeBox/NodeBox.tsx

```diff
@@ -2,13 +2,14 @@ import React from 'react'
 import PointerBox from '../PointerBox/PointerBox'
 import ValueBox from '../ValueBox/ValueBox'
 import css from 'styled-jsx/css'
-// TODO: Make files structure consistent (here ..Arrow/index.tsx)
 import Arrow from '../Arrow/Arrow'
+// TODO: Make files structure consistent (here ..Arrow/index.tsx)
```

Comment:
please rename those files and take a look at the whole project and make it consistent everywhere